docs/json/radarr/cf/dubs-only.json

```diff
@@ -12,7 +12,7 @@
       "negate": false,
       "required": false,
       "fields": {
-        "value": "(?<!multi-)\\b(dub(bed)?)\\b|(funi|eng(lish)?)_?dub"
+        "value": "^(?!.*Dual[-_. ]?Audio).*((?<!multi-)\\b(dub(bed)?)\\b|(funi|eng(lish)?)_?dub)"
```

Comment:
change the regex to "^(?!.*(Dual|Multi)[-_. ]?Audio).*((?<!multi-)\\b(dub(bed)?)\\b|(funi|eng(lish)?)_?dub)" to handle the mutli-audio dub cases (Arg0 Solo Levelling example)

Reply:
Fixed